atlascope/core/models/tour.py

```diff
@@ -0,0 +1,33 @@
+# from typing import List
+from uuid import uuid4
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.gis.db import models as gisModels
+
+from django.db import models
+from rest_framework import serializers
+
+
+class Tour(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    investigation = models.ForeignKey(
+        'Investigation',
+        on_delete=models.CASCADE,
+        related_name='tour',
+    )
+    way_points = ArrayField(
+        ArrayField(
+            gisModels.PointField(null=True),
+            null=True
+        ),
+        null=True
+    )
```

Comment:
Why a 2D array of points?

Two thoughts:
1. Ideally we'd create a `Waypoint` class (with a `Point` and an integer zoom level) that could be the field type, though I don't really know the right way to do that sort of thing.
2. It looks like `PointField` supports 3D points by default; we could certainly abuse that to represent a 2D location plus a zoom level.

Reply:
resolved with 56d7e61af852626496a99eb9a5c4ec4b7b136f08